url.bs

```diff
@@ -2403,16 +2402,12 @@ takes a byte sequence <var>input</var>, and then runs these steps:
 
    <li><p>Replace any 0x2B (+) in <var>name</var> and <var>value</var> with 0x20 (SP).
 
-   <li><p>Add a tuple consisting of <var>name</var> and <var>value</var> to <var>tuples</var>.
-  </ol>
-
- <li><p>Let <var>output</var> be an empty list of name-value tuples where both name and value hold a
- string.
+   <li><p>Let <var>nameString</var> and <var>valueString</var> be the result of running <a>UTF-8
+   decode without BOM</a> on the <a lt="percent decode">percent decoding</a> of <var>name</var> and
+   <var>value</var>, respectively.
 
- <li><p>For each name-value tuple in <var>tuples</var>, append a name-value tuple to
- <var>output</var> where the new name and value appended to <var>output</var> are the result of
- running <a>UTF-8 decode without BOM</a> on the <a lt="percent decode">percent decoding</a> of the
- name and value from <var>tuples</var>, respectively, using <var>encoding</var>.
+   <li><p><a for=list>Append</a> <var>nameString</var>/<var>valueString</var> to <var>output</var>.
```

Comment:
Could you use the tuple syntax instead? See https://github.com/whatwg/infra/issues/127.